src/App.js

```diff
@@ -0,0 +1,19 @@
+import { NavBar } from './components/navBar'
+import { React } from 'react'
+import './App.css'
+import { Routes, Route } from "react-router-dom";
+import {ProductDetails, Cart, WishList, Products} from './Pages/index'
+
+export default function App() {
+  return (
+    <div classNameName="App">
```

Comment:
classNameName is wrong. if not giving any styles to App class then remove and use react fragment.

Reply:
done